base/core/src/message_to_journeys.cc

```diff
@@ -47,6 +47,14 @@ journey::transport to_transport(Walk const& walk, uint16_t duration) {
   return t;
 }
 
+static uint32_t to_color(std::string_view const color_str) {
+  if (color_str.size() != 6) {
```

Comment:
Does it make sense to also check `std::all(..., isdigit)`?

Reply:
The input is a hex number, so that alone wont work, but I can certainly make this more robust (and same in nigiri, where this is probably even more important).